version.json

```diff
@@ -0,0 +1,3 @@
+{
+  "version": "v0"
+}
```

Comment:
Ah, we're using a different release process. We might want to delete this, `releaser.yml` and `release-check.yml`.

Reply:
And `tagpush.yml`.